cypress/integration/tests/administrator/credentials/remove_creds_app.test.ts

```diff
@@ -41,7 +41,7 @@ describe("Validation of Source Control Credentials", { tags: "@tier1" }, () => {
             getRandomAnalysisData(this.analysisData)
         );
         application.create();
-        application.manageCredentials(source_credential.name);
+        application.manageCredentials(source_credential);
```

Comment:
@ibragins This test need not be added in a separate test file . 
We can rename "sourceControl_crud.test.ts" to "sourcecontrol.test.ts"  and add it in that test file .